packages/nx/src/executors/run-commands/run-commands.impl.ts

```diff
@@ -208,6 +216,7 @@ function createProcess(
     const processExitListener = () => childProcess.kill();
     process.on('exit', processExitListener);
     process.on('SIGTERM', processExitListener);
+    process.on('SIGINT', () => childProcess.kill('SIGINT'));
```

Comment:
Lets reuse the `childProcessExitListener` here

Reply:
the problem is the exit listener won't forward the signal used to kill the main process. I can parameterize the `processExitListener` but I don't know if it's worth it?